paddle_fl/core/strategy/fl_strategy_base.py

```diff
@@ -146,7 +162,7 @@ def minimize(self, optimizer=None, losses=[]):
         """
         Define Dpsgd optimizer
         """
-        optimizer = fluid.optimizer.Dpsgd(self._learning_rate, clip=self._clip, batch_size=self._batch_size, sigma=self._sigma)
+        # optimizer = fluid.optimizer.Dpsgd(self._learning_rate, clip=self._clip, batch_size=self._batch_size, sigma=self._sigma)
```

Comment:
why do you comment this?

Reply:
Thanks. I have commented this for other purpose. I have uncommented this line.